packages/metaconfig/versions/0.17.x.js

```diff
@@ -13,6 +15,18 @@ class ZeroSeventeen {
       this.kind = 'service'
     }
   }
+
+  up () {
+    const original = this.config
+    const config = rfdc(original)
+    config.$schema = 'https://platformatic.dev/schemas/v0.18.0/db'
```

Comment:
Is `/db` supposed to be hard coded? Shouldn't it be `kind`?

Reply:
good spot!